ABloom/ABloom/Resources/Components/QnAListItem.swift

```diff
@@ -15,44 +15,45 @@ struct QnAListItem: View {
   
   var body: some View {
     HStack(spacing: 0) {
-      // TODO: 이미지 쉐도우 확인
+      
       Image(categoryImg)
         .resizable()
-        .frame(width: 50, height: 50)
         .aspectRatio(contentMode: .fit)
-        .overlay(
-          RoundedRectangle(cornerRadius: 36)
+        .frame(width: 50, height: 50)
+        .background(
+          RoundedRectangle(cornerRadius: 17)
             .clayMorpMDShadow()
-            .foregroundStyle(.clear)
-            .frame(width: 50, height: 50)
+            .foregroundStyle(.white)
         )
-        .frame(width: 50, height: 50)
         .padding(.trailing, 15)
       
-      
       VStack(spacing: 4) {
         HStack {
           Text(question)
             .fontWithTracking(.caption1Bold)
             .foregroundStyle(.black)
+          
           Spacer()
         }
+        
         HStack {
           Text(date)
             .fontWithTracking(.caption2R)
+          
           Spacer()
+          
           Text(isAns ? "" : "답변을 기다리고 있어요 >")
             .fontWithTracking(.caption2R)
         }
         .foregroundStyle(.stone500)
+        
       }
       Spacer()
-      
     }
     .padding([.leading, .trailing], 20)
```

Comment:
```suggestion
    .padding(.horizontal, 20)
```
수정되면 좋을 것 같습니다!

이부분도 상위뷰에서 패딩을 주는 방식을 고려해봐도 좋을 듯 합니다 🌝